pubspec.yaml

```diff
@@ -32,7 +32,7 @@ flutter:
       ios:
         pluginClass: MapboxMapsPlugin
       web:
-        default_package: mapbox_gl_web
+        default_package: maplibre_gl_web
```

Comment:
this seems to not change anything, but I think it was weird that we still referenced the old name before, and I could see that failing eventually.